src/sentry/ingest/consumer/processors.py

```diff
@@ -16,13 +17,15 @@
 from sentry.models.project import Project
 from sentry.signals import event_accepted
 from sentry.tasks.store import preprocess_event, save_event_feedback, save_event_transaction
+from sentry.usage_accountant import record
 from sentry.utils import json, metrics
 from sentry.utils.cache import cache_key_for_event
 from sentry.utils.dates import to_datetime
 from sentry.utils.snuba import RateLimitExceeded
 
 logger = logging.getLogger(__name__)
 
+RESOURCE_ID = "rc_processing_redis"
```

Comment:
Config option (runtime option). No. There is no need. This will never change at runtime.
A setting (werver.py) maybe, but it is not very useful if this changed between environments. 

Will move it there anyway.